eZ/Bundle/EzPublishCoreBundle/Translation/Collector.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * File containing the Collector class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Translation;
+
+/**
+ * Retrieves all installed ezplatform translation files.
+ */
+class Collector
```

Comment:
Well... I did stumble upon the "vendor" pattern question. It is a good one, actually, but it has implications.

I'd still go for "GlobSomething" as the implementation, as it is an implementation that uses, well, `glob()` 👿.

But what about a `PatternCollector` ? Then it would have a `setPattern` method, so that the implementation can be given a pattern. We can still do a `db`, or `glob`, implementation.